deps/v8/gypfiles/v8.gyp

```diff
@@ -361,6 +361,18 @@
             '../src/builtins/builtins-intl-gen.cc',
           ],
         }],
+        # Platforms that don't have CAS support need to link atomic library
```

Comment:
I realize this is copied from the upstream comment, but it might help to spell out CAS?

Reply:
Ack